index.html

```diff
@@ -38,35 +39,220 @@ <h1 class="name">I’m Abraha Glad to see you!</h1>
+      </div>
+      <!-- the fourth work -->
+      <div class="card">
+        <img
+          class="image1"
+          src="images/Snapshoot-Portfolio(4).svg"
+          alt="Porfolio-image1"
+        />
+        <div class="headline2">
+          <ul class="tonic">
             <li>
-              <a href="#"><i class="fa fa-angellist fa-lg"></i></a>
+              <h3>
+                Multi-Post <br />
+                Stories
+              </h3>
             </li>
           </ul>
+          <ul class="card-head">
+            <li class="canopy1">CANOPY</li>
+            <li class="canopy">Back End Dev</li>
+            <li class="canopy">2015</li>
+          </ul>
+        </div>
+        <p>
+          A daily selection of privately personalized reads; no accounts or
+          sign-ups required.
+        </p>
+        <ul class="cardsub">
+          <li class="list">html</li>
+          <li class="list">css</li>
+          <li class="list">javaScript</li>
+        </ul>
+        <button type="button">See Project</button>
+      </div>
+    </section>
+    <!-- About myself -->
+    <section class="personal-info">
+      <div class="about-me">
+        <h2>
+          About <br />
+          Myself
+        </h2>
+        <p>
+          Hello I’m a software developer! I can help you build a product ,
+          feature or website Look through some of my work and experience! If you
+          like what you see and have a project you need coded, don’t hestiate to
+          contact me.
+        </p>
+        <div class="social-container">
+          <a href="#">LET'S CONNECT</a>
+          <div class="social-media2">
+            <ul>
+              <li>
+                <a href="https://twitter.com/Alamsbut"
+                  ><i class="fa fa-twitter fa-lg"></i
+                ></a>
+              </li>
+              <li>
+                <a href="https://www.linkedin.com/in/abraha-kahsay-492771135/"
+                  ><i class="fa fa-linkedin fa-lg"></i
+                ></a>
+              </li>
+              <li>
+                <a href="#"><i class="fa fa-medium fa-lg"></i></a>
+              </li>
+              <li>
+                <a href="https://github.com/AbrahaKahsay"
+                  ><i class="fa fa-github fa-lg"></i
+                ></a>
+              </li>
+              <li>
+                <a href="#"><i class="fa fa-angellist fa-lg"></i></a>
+              </li>
+            </ul>
+          </div>
+        </div>
+        <div class="btn">
+          <button type="button">Get my resume</button>
         </div>
+        <ul>
+          <li class="langguages">
+            Langguages
+            <ul>
+              <li><img src="images/javaScript.svg" alt="javaScript" /></li>
+              <li><img src="images/html.svg" alt="html" /></li>
+              <li><img src="images/css.svg" alt="css" /></li>
+            </ul>
+          </li>
+          <li>Frameworks</li>
+          <li>Skills</li>
```

Comment:
- [ ] [REQUIRED] As stated in the project requirements, your design should match the original design, please add the right arrow buttons on these two sections _'Frameworks and Skills'_. and add the down arrow button on the _'languages'_ section
- [ ] Remember to include the horizontal lines separating the two segments
### Your Design
![stories4](https://user-images.githubusercontent.com/63964646/149623925-8da760dd-7bf8-4180-a713-a2e97825b77b.PNG)

### Original Design

![stories3](https://user-images.githubusercontent.com/63964646/149623913-914553f4-36fc-4c19-84b4-7de3461c8c83.PNG)

Reply:
Great job implementing this 👍🏽